### PR TITLE
gallery-dl: add page

### PR DESCRIPTION
**Pull request by rail01**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.25.4

### CODE REVIEW

File: pages/common/gallery-dl.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# gallery-dl
+
+> Download image galleries and collections from several image hosting sites.
+> More information: <https://github.com/mikf/gallery-dl>.
+
+- Download images; in this case from danbooru via tag search for 'bonocho':
```
Comment (acuteenvy) [inline pages/common/gallery-dl.md:6]:
```suggestion
- Download images from the specified URL:
```

File: pages/common/gallery-dl.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`gallery-dl "https://danbooru.donmai.us/posts?tags=bonocho"`
```
Comment (acuteenvy) [inline pages/common/gallery-dl.md:8]:
```suggestion
`gallery-dl "{{url}}"`
```

File: pages/common/gallery-dl.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Get the direct URL of an image from a site supporting authentication with username & password:
```
Comment (acuteenvy) [inline pages/common/gallery-dl.md:10]:
```suggestion
- Get the direct URL of an image from a site supporting authentication with username and password:
```

File: pages/common/gallery-dl.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`gallery-dl -g -u "<username>" -p "<password>" "https://twitter.com/i/web/status/604341487988576256"`
```
Comment (acuteenvy) [inline pages/common/gallery-dl.md:12]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax
```suggestion
`gallery-dl --get-urls --username {{username}} --password {{password}} "{{url}}"`
```

File: pages/common/gallery-dl.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Filter manga chapters by chapter number and language:
+
+`gallery-dl --chapter-filter "10 <= chapter < 20" -o "lang=fr" "https://mangadex.org/title/59793dd0-a2d8-41a2-9758-8197287a8539"`
```
Comment (acuteenvy) [inline pages/common/gallery-dl.md:16]:
```suggestion
`gallery-dl --chapter-filter "{{10 <= chapter < 20}}" --option "lang={{language_code}}" "https://mangadex.org/title/59793dd0-a2d8-41a2-9758-8197287a8539"`
```

**Comment by rail01** [comment]
@acuteenvy Thanks for improvements!

File: pages/common/gallery-dl.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+`gallery-dl "{{url}}"`
+
+- Get the direct URL of an image from a site supporting authentication with username and password:
+
+`gallery-dl --get-urls --username {{username}} --password {{password}} "{{url}}"`
+
+- Filter manga chapters by chapter number and language:
+
+`gallery-dl --chapter-filter "{{10 <= chapter < 20}}" --option "lang={{language_code}}" "https://mangadex.org/title/59793dd0-a2d8-41a2-9758-8197287a8539"`
```
Comment (kbdharun) [inline pages/common/gallery-dl.md:16]:
```suggestion
`gallery-dl --chapter-filter "{{10 <= chapter < 20}}" --option "lang={{language_code}}" "{{url}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**